nav2_collision_monitor/README.md

```diff
@@ -66,3 +67,9 @@ The following notes could be made:
 
  * Due to sheer speed, circle shapes are preferred for the approach behavior models if you can approximately model your robot as circular.
  * More points mean lower performance. Pointclouds could be culled or filtered before the Collision Monitor to improve performance.
+
+
+## Collision Detector
```

Comment:
Also, in the beginning of the `Collision Detector` section, it could be great to add a few words about background/prerequisites for creating CD: why it is was introduced (e.g. CM only changes robot's behavior, but we might need to have an information utilized either by external module or user) and what examples of use-cases CD to cover (e.g. LED on/off, other safety devices connection, obstacle detection alarm, connection with other robots/modules, etc...)

Reply:
I added a little more verbosity but since it's similar to collision monitor in many aspects (as stated), I'm not sure what more to add. Please give specific examples if you think it's not enough.